src/main/java/de/uniwue/helper/TrainingHelper.java

```diff
@@ -198,10 +198,10 @@ public void execute(List<String> cmdArgs, String projectName, String trainingId)
         writer.writeValue(segmentListFile, segmentObj); 
         command.add(segmentListFile.toString());
         System.out.println(segmentListFile);
-        
```

Comment:
If possible no unneeded line changes (To prevent more or less unchanged files to appear in the diff)